source/_integrations/nextdns.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: NextDNS
+description: Instructions on how to integrate NextDNS within Home Assistant.
+ha_category:
+  - Network
+ha_release: 2022.8.0
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@bieniu'
+ha_domain: nextdns
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+NextDNS is a DNS service that protects from all kinds of security threats, blocks ADS and trackers on websites and in apps and provides a safe and supervised Internet for kids — on all devices and on all networks. The NextDNS integration allows you to monitor your NextDNS statistics.
+
+## Setup
```

Comment:
Small change as this is how we generally reference this section in our documentation.

```suggestion
## Prerequisites
```